assets/cms/js/vue/Manager.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable no-new */
+/* global Vue */
+import Context from './Context'
+
+class Manager {
+    /**
+     * Ensures that our Concrete.Vue manager is available on the window object.
+     * Note: Do NOT call this before the global Concrete object is created in the CMS context.
```

Comment:
We should make it so that loading this before loading the global concrete object works fine.

We can do that by adding: `...(global.Concrete || {}),` to the top of this object declaration here: https://github.com/concrete5/bedrock/blob/5858bcd48bac394ffb283e4d95d52e3e10452d35/assets/cms/js/concrete5.js#L26
It's probably worth doing that here so that we can support loading vue anywhere